<ui version="4.0" >
 <class>NeuesKontoDialog</class>
 <widget class="QWidget" name="NeuesKontoDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>create new account</string>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="labelKontoName" >
      <property name="text" >
       <string>name of account:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="lineKontoName" />
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="labelKontoBeschreibung" >
      <property name="text" >
       <string>description of account:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="lineKontoBeschreibung" />
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="labelBLZ" >
      <property name="text" >
       <string>bank identification number:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLineEdit" name="lineBLZ" />
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="labelBankname" >
      <property name="text" >
       <string>name of bank:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QLineEdit" name="lineBankName" />
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="labelLimit" >
      <property name="text" >
       <string>limit:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QDoubleSpinBox" name="spinboxLimit" >
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="prefix" >
       <string/>
      </property>
      <property name="suffix" >
       <string>â‚¬</string>
      </property>
      <property name="minimum" >
       <double>-99999.000000000000000</double>
      </property>
      <property name="maximum" >
       <double>99999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>can be under limit:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" >
     <widget class="QCheckBox" name="checkBeUnderLimit" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>10</y>
     <width>77</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QPushButton" name="pushAdd" >
      <property name="text" >
       <string>add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushEnd" >
      <property name="text" >
       <string>close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushEnd</sender>
   <signal>clicked()</signal>
   <receiver>NeuesKontoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>510</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
